### PR TITLE
report(redesign): design review feedback

### PR DESCRIPTION
**Pull request by Hoten**
#8185

See feedback under "Round two": https://docs.google.com/document/d/15L5iaWo4ieouegpSlaqGGWpA13JnTv73xnpxuVhzkBg/edit

See commit descriptions for whats up in this PR

### CODE REVIEW

File: lighthouse-core/report/html/renderer/report-ui-features.js [hunk 1]
```diff
@@ -120,17 +119,6 @@ class ReportUIFeatures {
     this.onMediaQueryChange(mediaQuery);
   }
 
-  _setupSmoothScroll() {
-    for (const el of this._dom.findAll('a.lh-gauge__wrapper', this._document)) {
-      const anchorElement = /** @type {HTMLAnchorElement} */ (el);
-      anchorElement.addEventListener('click', e => {
-        e.preventDefault();
-        window.history.pushState({}, '', anchorElement.hash);
-        this._dom.find(anchorElement.hash, this._document).scrollIntoView({behavior: 'smooth'});
```
Comment (paulirish) [inline lighthouse-core/report/html/renderer/report-ui-features.js]: wait why?

Comment (Hoten) [reply lighthouse-core/report/html/renderer/report-ui-features.js]: was mentioned in the design meeting. @yuinchien asked if the animation could be shorter. i said what if it were just removed completely. i thought everyone said ok

idea was just - user wants to see a particular part of the report, so just show them as soon as intent is made w/o the fanfare of a scroll animation.

Comment (paulirish) [reply lighthouse-core/report/html/renderer/report-ui-features.js]: yuin and i are still attached to it. i got a branch for fixing the highlighter anim

File: lighthouse-core/report/html/report-styles.css [hunk 2]
```diff
@@ -1158,18 +1164,18 @@
...
   --image-preview-size: 24px;
   border-collapse: collapse;
+  /* Can't assign padding to table, so shorten the width instead. */
+  width: calc(100% - var(--inner-audit-left-padding));
 }
 
-.lh-table thead {
-  background-color: var(--lh-table-higlight-bg);
-}
 .lh-table thead th {
```
Comment (paulirish) [inline lighthouse-core/report/html/report-styles.css]: we had something in there about text alignment with headers


image

1. getting description flush left with the title.
1. getting URL flush left with those two.

right, @yuinchien?

Comment (Hoten) [reply lighthouse-core/report/html/report-styles.css]: hm i aligned it to the description instead

Comment (yuinchien) [reply lighthouse-core/report/html/report-styles.css]: @Hoten the title, description, the table heading(URL), the table should all align left to the same line. 

Screen Shot 2019-05-03 at 1 47 23 PM

Comment (yuinchien) [reply lighthouse-core/report/html/report-styles.css]: @Hoten can we horizontally center the shape (triangle) with the first line of title? Thanks!
Screen Shot 2019-05-03 at 1 48 48 PM

Comment (Hoten) [reply lighthouse-core/report/html/report-styles.css]: done

**Comment by Hoten** [comment]
todo
image

**Comment by paulirish** [comment]
oops. one more piece on the alignment it seems:

image

the left padding on the th:first-child should be 0 to get "URL" flush with the rest.
and with right paddign on th:last-child

**Comment by Hoten** [comment]
> the left padding on the th:first-child should be 0 to get "URL" flush with the rest.
done

File: lighthouse-core/report/html/renderer/report-ui-features.js (hunk 1 above)
Comment (brendankenny) [reply lighthouse-core/report/html/renderer/report-ui-features.js]: @paulirish whats up with this

**Pull request merged**